sys-libs/libmodulemd/libmodulemd-2.14.0.ebuild

```diff
@@ -0,0 +1,58 @@
+# Copyright 2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{9..11} )
+
+inherit meson python-single-r1
+
+DESCRIPTION="C Library for manipulating Fedora Modularity metadata files"
+HOMEPAGE="https://github.com/fedora-modularity/libmodulemd"
+if [[ ${PV} = 9999* ]]; then
+		inherit git-r3
+		EGIT_REPO_URI="https://github.com/fedora-modularity/libmodulemd.git"
+else
+		SRC_URI="https://github.com/fedora-modularity/libmodulemd/archive/refs/tags/${PV}.tar.gz -> ${P}.tar.gz"
+		KEYWORDS="~amd64"
+fi
+
+LICENSE="MIT"
+SLOT="0"
+
+IUSE="gtk-doc test"
+RESTRICT="!test? ( test )"
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+DEPEND="
+	${PYTHON_DEPS}
+	app-arch/rpm
+	sys-apps/file
+	dev-libs/glib:2
+	dev-libs/libyaml
+	$(python_gen_cond_dep '
+		dev-python/pygobject:3[${PYTHON_USEDEP}]
+		')
+"
+RDEPEND="${DEPEND}"
+BDEPEND="
+	${PYTHON_DEPS}
+	dev-libs/gobject-introspection
+	gtk-doc? (
+		dev-libs/glib[gtk-doc]
+		dev-util/gtk-doc
+	)
+	test? (
+		sys-libs/libmodulemd
+		dev-util/valgrind
+	)
+"
+
+src_configure() {
+	local emesonargs=(
+		$(meson_use gtk-doc with_docs)
+		$(meson_use test test_installed_lib)
+	)
+	# There isn't an option to skip valgrind tests at this point, and the whole test suite takes 7s.
```

Comment:
This is _tolerable_ for now if you must, but I prefer to not run Valgrind at all for tests, because it's not portable to other arches.